### DISCUSSION TITLE
Information about the lifecycle of cli releases.

### DISCUSSION THREAD

**Discussion by krishnayadav2-tf** [forcedotcom/cli#3129] [General]
Hi , 
I wanted to understand about the lifecycle of the cli releases being followed here. 
Is there any convention followed as Active , LTS for the cli releases. 
Can't see any timelines in the release notes. Hence thought to open up a discussion thread here. 
Any useful information will be much appreciated.

**Comment by WillieRuemmele** [comment]
Hi @krishnayadav2-tf -

we use a `nightly` -> `latest-rc` -> `latest` process,

so `nightly` contains the absolute latest code, that we've published, as of the day previous

on wednesdays, we take the last 7 `nightly`'s and create a `latest-rc` release, this is what will be promoted to `latest` in one weeks' time, so this is more stable than nightly, but there can still be bugs in here. We do this so that the community can see what we're releasing next week, play with it, find any bugs, report them, comment on behaviors, etc...

and the next wednesday, we take the `latest-rc` build, and promote that to `latest`, where it will be supported for the next week
